### DISCUSSION TITLE
How to use CRL on leaf certificates

### DISCUSSION THREAD

**Discussion by sc-sityad** [smallstep/certificates#1422] [Q&A] [answered]
-  How to get CRL http URL to store the ca.url. I followed all the steps mentioned in  ENABLE ACTIVE REVOCATION ON YOUR INTERMEDIATE CA. 
- I have enable the crl in ca.json `"crl": {
	    "enabled": true,
	    "generateOnRevoke": true
	}`
- I create a certificate using root_ca.crt using following command `step ca certificate emqx emqx.crt emqx.key  --kty RSA`.
- Now I want to revoke it using comand. `step ca revoke --cert emqx.crt --key emqx.key`. 
- The certificate is revoked but how the CRLs is updated in this case, I am not able to understand.
- What is the use of index.txt, Do I have to manually add the revoked data in that file?
- How the whole CRL support works in this case?

**Comment by maraino** [comment, marked as answer]
I've responded to this here: https://github.com/smallstep/certificates/issues/1423#issuecomment-1581568312
